provisions/roles/jenkins/master/tasks/main.yml

```diff
@@ -2,22 +2,39 @@
   tags:
   - jenkins/master
 
+# Stop Jenkins so no files are overriden by a running Jenkins instance
+- name: Stop jenkins before configuration
```

Comment:
this was being taken care of by the cleanup task though.

Reply:
Yeah.. Problem I was facing was that configurations were *still* being overriden unless I had this added in.